test/integration/policyOps.test.js

```diff
@@ -201,19 +201,28 @@ lab.experiment('PolicyOps', () => {
           name: 'called',
           description: 'called',
           organizationId: orgId,
-          policies: [{
-            key: 'userPolicy'
-          }, {
-            key: 'policyWithoutVariables'
-          }, {
-            key: 'policyWithVariables',
-            variables: {var1: 'value1'}
-          }, {
-            key: 'policyWithVariablesMulti',
-            variables: {var2: 'value2'}
-          }, {
-            key: 'sharedPolicy'
-          }]
+          policies: [
```

Comment:
Would you do me the favour of "duplicating"  `policyWithVariables` rather than `policyWithVariablesMulti` (btw to me this name kind of suggested some kind of feature... which is not the case 😕 )

For someone reading the tests for policies with variables for the first time, I must say that the names and values chosen lean more towards confusion and not clarity. But no need to take action If you don't feel like 😄